supervisor.conf.j2

```diff
@@ -6,3 +6,8 @@ environment=DJANGO_SETTINGS_MODULE="{{ project_name }}.settings.{{ django_settin
 
 [group:{{ project_name }}]
 programs=gunicorn-{{ project_name }}
```

Comment:
Looking at this now. It seems to be failing with exceptions like the following on test:

File "/opt/tlt/lti_emailer/manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/huey/djhuey/management/commands/run_huey.py", line 116, in handle
    self.autodiscover()
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/huey/djhuey/management/commands/run_huey.py", line 93, in autodiscover
    self.autodiscover_appconfigs()
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/huey/djhuey/management/commands/run_huey.py", line 63, in autodiscover_appconfigs
    app_path = config.module.__path__
AttributeError: 'module' object has no attribute '__path__'